EventListener/PjaxBrowserLanguageSubscriber.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * On PJAX requests, sets the request's locale from the browser's accept-language header.
+ */
+class PjaxBrowserLanguageSubscriber implements EventSubscriberInterface
+{
+    public static function getSubscribedEvents()
+    {
+        return [
+            KernelEvents::REQUEST => [
+                // priority needs to be higher than the CoreBundle's LocaleListener
+                ['setPjaxRequestLocale', 50],
+            ],
+        ];
+    }
+
+    public function setPjaxRequestLocale(GetResponseEvent $event)
+    {
+        $request = $event->getRequest();
+
+        if (!$event->isMasterRequest() || !$request->headers->has('x-pjax')) {
```

Comment:
Nope, we do not. As said, while I don't disagree with the usage of constants for this, I don't have a good place for it at the moment. It is also used in another listener, and therefore can't go into this listener. The PJAX controllers do not share any common class that we could use for this.

Would you create an interface or class for this ? I'd suggest that we postpone this for when we will work further on PJAX, and find a suitable place for the constant at this point. What do you think ?